data_analysis/profit_analysis.py

```diff
@@ -4,17 +4,39 @@
 """
 
 
-def sell_today(value_today, value_tomorrow, no_units, daily_cost):
+def sell_today(value_today, value_tomorrow, no_units, daily_cost, error_tomorrow_upper,
+               error_tomorrow_lower):
```

Comment:
This wont pass the unit tests now, i'll do a PR for it because this is the only function we have a test for lol